lib/engine/game/g_18_zoo/step/buy_train.rb

```diff
@@ -62,10 +62,10 @@ def process_buy_train(action)
             @game.first_train_of_new_phase = false
           end
 
-          def sellable_bundle?(bundle)
```

Comment:
using just single item bundle was breaking the text for "you can get $N selling shares", so designer decided we can ignore for online version